main.py

```diff
@@ -154,27 +156,55 @@ def searchBox(evt):
 
 def bookProcured():
     def addTrans():
-        addTotal = Label(addTransF, text="tot", bg="#FFFFFF", font = ("Berlin Sans FB", 12))
-        addTotal.grid(row=9, column=1)
+        Atotal = float(addBookPriceE.get())*int(addCopiesE.get())
+        Label(addTransF, text="total: ", bg="#FFFFFF", font = ("Berlin Sans FB", 12)).grid(row=9, column=1)
+        Label(addTransF, text=str(Atotal), bg="#FFFFFF", font = ("Berlin Sans FB", 12)).grid(row=9, column=2)
+        mycursor = mycon.cursor()
+        mycursor.execute(f"INSERT INTO fromven VALUES {int(addTransIDE.get())},{int(addSuppIDE.get())},{int(addBookIDE.get())},{int(addCopiesE.get())},{Atotal}")
+        mycon.commit()
+        mycursor.execute(f"INSERT INTO vendors VALUES {int(addSuppIDE.get())},{addSuppNameE.get()},{addContactE.get()} ON DUPLICATE KEY UPDATE")
+        mycon.commit()
         # clear all entries after adding
+        addBookPriceE.delete(0, END)
+        addBookIDE.delete(0, END)
+        addContactE.delete(0, END)
+        addSuppNameE.delete(0, END)
+        addCopiesE.delete(0, END)
+        addSuppIDE.delete(0, END)
+        addTransIDE.delete(0, END)
         return
 
     def modTrans():
+        Mtotal = int(modBookPriceE.get())*int(modCopiesE.get())
```

Comment:
float(modBookPriceE)? also is Mtotal being displayed? if yes, label is missing. if not, modTotal label is unnecessary.